Description: Upstream changes introduced in version 0:20090124-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 gant (0:20090124-1) UNRELEASED; urgency=low
 .
   * Initial release. Closes: #xxxxxx
 .
 The person named in the Author field signed this changelog entry.
Author: Christian Perrier <bubulle@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- gant-20090124.orig/Makefile
+++ gant-20090124/Makefile
@@ -10,4 +10,4 @@ gant.o:	gant.c antdefs.h
 antlib.o: antlib.c antdefs.h
 
 clean:
-	rm *.o gant
+	rm *.o gant || true
